src/platforms/unity/native-support.mdx

```diff
@@ -14,14 +14,38 @@ Native support requires configuration using the Sentry Unity Editor window. Supp
 
 Native support is `enabled` by default. The Unity SDK passes the options defined through the Sentry Unity Editor window during build time. For Android, it embeds the [Android SDK](/platforms/android/) inside the Gradle project created by Unity. The [iOS SDK](/platforms/apple/guides/ios/) is embedded inside the generated Xcode project. This means there's no need to use the Unity [built-in crash reporting](https://docs.unity3d.com/ScriptReference/CrashReport.html) functionality.
 
+You can opt out of native support by unchecking `iOS Native Support` and `Android Native Support` in the configuration window.
+
+## Debug Symbols
+
+Sentry requires [debugs information files](platforms/android/data-management/debug-files/) to symbolicate your crash logs. Our Unity SDK does not yet support the automated upload of debug symbols, but since it's using other Sentry SDKs this can be set up manually.
```

Comment:
```suggestion
Sentry requires [debug information files](/platforms/android/data-management/debug-files/) to symbolicate your crash logs. Our Unity SDK does not yet support the automated upload of debug symbols, but since it's using other Sentry SDKs, this can be set up manually.
```